src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -400,23 +401,6 @@ protected int doWork() {
                     UmiUtil.setMolecularIdentifier(rec, "", MOLECULAR_IDENTIFIER_TAG, DUPLEX_UMI);
                 }
 
-                // Tag any read pair that was in a duplicate set with the duplicate set size and a representative read name
-                if (TAG_DUPLICATE_SET_MEMBERS) {
```

Comment:
was this code really copy-pasted like this all along and nobody noticed? 😞

Reply:
yeah... that does appear to be the case